src/sentry/api/event_search.py

```diff
@@ -303,7 +302,7 @@ def handle_negation(negation, operator):
 
 def get_operator_value(operator):
     if isinstance(operator, Node):
-        operator = "=" if isinstance(operator.expr, Optional) else operator.text
+        operator = operator.text or "="
```

Comment:
This had to be changed because there were internals changed in the new version. From my experiments, if the matched text is `""` we can assume that the value was optional and not found.